pkg/webhook/webhook.go

```diff
@@ -162,15 +162,14 @@ func (w *Config) buildValidatingWebHooks() []admv1.ValidatingWebhook {
 	sideEffortsNone := admv1.SideEffectClassNone
 	namespacedScope := admv1.NamespacedScope
 	clusterScope := admv1.ClusterScope
-
+	webhookTimeoutSeconds := pointer.Int32(1)
```

Comment:
@ryanzhang-oss why is the timeout only set to 1 second ?

Reply:
That's the least we can set